include/realtime_tools/realtime_server_goal_handle.h

```diff
@@ -114,6 +125,10 @@ class RealtimeServerGoalHandle
         else
           gh_.setSucceeded();
       }
+      if (req_feedback_)
+      {
+        gh_.publishFeedback(*req_feedback_);
```

Comment:
What happens when `req_feedback_` is NULL as per the default value of `setFeedback(...)`?